lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -235,6 +239,22 @@ def call(env)
             env[:ui].info(" -- USB passthrough:   #{usb_dev.join(', ')}")
           end
 
+          @redirdevs.each do |redirdev|
+            redirdev_dev = []
```

Comment:
Have it changed to an output like the following. What do you think about this?

~~~
==> client-1:  -- Redirected Devices: 
==> client-1:     -> bus=usb, type=spicevmc
==> client-1:     -> bus=usb, type=spicevmc
==> client-1:     -> bus=usb, type=spicevmc
==> client-1:     -> bus=usb, type=spicevmc
==> client-1:  -- USB Device filter for Redirected Devices: 
==> client-1:     -> class=0x08, vendor=0x08e6, product=0x0437, version=2.00, allow=yes
==> client-1:     -> class=-1, vendor=-1, product=-1, version=-1, allow=no
~~~